src/Pages/ReferenceTokens.razor

```diff
@@ -71,21 +80,33 @@
                 </EditForm>
             </div>
             <div class="col-3">
-                <ReferenceTokenStatus Model="@_referenceTokenData"></ReferenceTokenStatus>
+                <ReferenceTokenStatus Model="@_referenceTokenData"/>
             </div>
         </div>
     </TabPane>
 </div>
+@if (RevokeDialogOpen)
```

Comment:
Could this `if` be moved into the `ModalDialog` component itself? It feels weird that as a user of the `ModelDialog` component I have to conditionally render it depending on whether we want it open or not. I'd rather have a boolean property on the `ModalDialog` component that controls if it is open or not?

Reply:
Moved it in 23ec94e